src/api/app/assets/javascripts/webui/application.js

```diff
@@ -207,7 +207,7 @@ function change_role(obj) {
         complete: function () {
             $('#' + type + '_spinner').hide();
             $('#' + type + '_table input').animate({opacity: 1}, 200);
-            $('#' + type + '_table input').removeAttr('disabled');
+            $('#' + type + '_table input').prop('disabled', false);
```

Comment:
I happend to learn the hard way :)

https://github.com/os-autoinst/openQA/commit/ac7fe8b2b5ac80325ef5266c8110929c09f383d7#diff-9fb9b9a3ce0b13423f7eb2ba327405b4